### PR TITLE
CORE-516 Trying to link botan with jitter

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: android/external/MakefileTemplate [hunk 1]
```diff
@@ -163,20 +163,22 @@ uuid/jni/Android.mk: uuid.src.stamp
 jitter-build: $(LOCAL)/lib/libjitterentropy.a
 
 $(LOCAL)/lib/libjitterentropy.a:
-	$(MAKE) -C $(JITTER_SRC) jitterentropy-static
-	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-includes PREFIX=$(LOCAL)
-	$(MAKE) -C $(JITTER_SRC) jitterentropy-static install-static PREFIX=$(LOCAL)
+	rsync -av $(JITTER_SRC)/ jitterentropy/
+	$(MAKE) -C jitterentropy AR="$(AR)" CC="$(CC)" CXX="$(CXX)" jitterentropy-static
+	$(MAKE) -C jitterentropy AR="$(AR)" CC="$(CC)" CXX="$(CXX)" jitterentropy-static install-includes PREFIX=$(LOCAL)
+	$(MAKE) -C jitterentropy AR="$(AR)" CC="$(CC)" CXX="$(CXX)" jitterentropy-static install-static PREFIX=$(LOCAL)
```
Comment (ghost) [inline android/external/MakefileTemplate]: jitterentropy-static can go away, it has already been built

File: android/jni/Android.mk [hunk 2]
```diff
@@ -17,10 +17,16 @@ LOCAL_MODULE := libpep_engine_sequoia_backend
...
+include $(CLEAR_VARS)
+LOCAL_MODULE := jitterentropy
+LOCAL_SRC_FILES := $(GPGBUILD)/$(TARGET_ARCH_ABI)/lib/libjitterentropy.a
+include $(PREBUILT_STATIC_LIBRARY)
+
 $(warning ==== JNIADAPTER android.mk using BOTAN2 for pEpEngineSequoiaBackend)
 include $(CLEAR_VARS)
 LOCAL_MODULE := botan
 LOCAL_SRC_FILES := $(GPGBUILD)/$(TARGET_ARCH_ABI)/lib/libbotan-2.a
+LOCAL_CPPFLAGS += -O3
```
Comment (ghost) [inline android/jni/Android.mk]: Not sure that is needed, because botan gets configured explicitly anyways with its flags. But I guess you tested that.

**Review by ghost — CHANGES_REQUESTED** [review]

File: android/jni/Android.mk (hunk 2 above)
Comment (ig-garcia) [reply android/jni/Android.mk]: not sure if there is any difference to be honest as the times range always from 15 to 40 seconds.

File: android/external/MakefileTemplate (hunk 1 above)
Comment (ig-garcia) [reply android/external/MakefileTemplate]: Removed.

File: android/jni/Android.mk (hunk 2 above)
Comment (ig-garcia) [reply android/jni/Android.mk]: Removed.

**Review by ghost — APPROVED** [review]

**Pull request merged**